monitor/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2.4'
+services:
+  monitor:
+    build:
+      context: ..
+      dockerfile: monitor/Dockerfile
+    ports:
+    - "${PORT}:${PORT}"
```

Comment:
We need to specify in README that the web interface is run on `PORT`. And extend the example of the configuration file with this parameter.

Reply:
Right. The `.env.example` already has this variable.
I updated the readme adding info about the web interface running on `PORT`